sycl/test-e2e/TaskSequence/concurrent-loops.cpp

```diff
@@ -7,9 +7,9 @@
 //===----------------------------------------------------------------------===//
 
 // FIXME: failure in post-commit, re-enable when fixed:
-// UNSUPPORTED: linux
+// XFAIL: *
```

Comment:
Our LIT infrastructure changes XFAIL to UNSUPPORTED, so these tests will always be skipped, and won't be even executed, so we won't know when the tests started to pass. @bowenxue-intel if it is ok for you, we can proceed to merging this